base/src/main/java/com/instaclustr/cassandra/ldap/conf/LdapAuthenticatorConfiguration.java

```diff
@@ -53,6 +53,9 @@ public final class LdapAuthenticatorConfiguration
 
     public static final String CASSANDRA_AUTH_CACHE_ENABLED_PROP = "auth_cache_enabled";
 
+    // allow as default or disallow empty passwords when try to connect to ldap server - empty passwords make do some unexpected behavior
+    public static final String ALLOW_BLANK_PASSWORD_PROP = "allow_blank_password";
```

Comment:
do you mean _blank_ or _empty_ ? A blank string is also this ` ` - that means a string consisting of a space. A blank string and an empty string are two different things.

Reply:
yes, i mean empty password - only zero length string, with blank password authentication will be failed as it is expecting